src/Storage/Storage.Management.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -1289,4 +1289,46 @@ function Test-NewSetAzureStorageAccount_LargeFileShare
         # Cleanup
         Clean-ResourceGroup $rgname
     }
+}
+
+<#
+.SYNOPSIS
+Test Test-NewAzureStorageAccountQueueTableEncrytionKeyType
+.DESCRIPTION
+SmokeTest
+#>
+function Test-NewAzureStorageAccountQueueTableEncrytionKeyType
+{
+    # Setup
+    $rgname = Get-StorageManagementTestResourceName;
+
+    try
+    {
+        # Test
+        $stoname = 'sto' + $rgname;
+        $stotype = 'Standard_LRS';
+        $loc = Get-ProviderLocation_Canary ResourceManagement;
+        $kind = 'StorageV2'
+
+        New-AzResourceGroup -Name $rgname -Location $loc;
+        Write-Output ("Resource Group created")
+		
+		# new account
+        New-AzStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -SkuName $stotype -EncryptionKeyTypeForTable Service -EncryptionKeyTypeForQueue Service
+
+        Retry-IfException { $global:sto = Get-AzStorageAccount -ResourceGroupName $rgname -Name $stoname; }
+        Assert-AreEqual $stoname $sto.StorageAccountName;
+        Assert-AreEqual $stotype $sto.Sku.Name;
+        Assert-AreEqual $loc.ToLower().Replace(" ", "") $sto.Location;
+        Assert-AreEqual $kind $sto.Kind;
+        Assert-AreEqual "Account" $sto.Encryption.Services.Blob.KeyType
+        Assert-AreEqual "Account" $sto.Encryption.Services.File.KeyType
```

Comment:
Should we check for `$sto.Encryption.Services.Table.KeyType` and `$sto.Encryption.Services.Queue.KeyType`?

Reply:
You are right, I will fix it soon

Reply:
Anyway, since the test doesn't block the binary build, for the time limitation, how about merge the PR first, and build the RC. I will raise another PR for the test fix.